components/tabs/Tabs.jsx

```diff
@@ -14,18 +13,23 @@ const propTypes = {
     types: PropTypes.arrayOf(PropTypes.shape({
         label: PropTypes.string.isRequired,
         internal: PropTypes.string.isRequired,
-        labelContent: PropTypes.element
+        count: PropTypes.number,
+        disabled: PropTypes.bool,
+        tooltip: PropTypes.element,
+        count: PropTypes.number
     })).isRequired,
     active: PropTypes.string.isRequired,
     switchTab: PropTypes.func.isRequired,
-    tabsClassName: PropTypes.string
+    tabsClassName: PropTypes.string,
+    tablessStyle: PropTypes.bool
```

Comment:
Not gonna die on this hill but would prefer this to be an enum. like `tabType` maybe?

Reply:
I think `tablessStyle` is either 0 or 1 or true or false and why it is a boolean. In the future if we have more tab styles we can apply different classnames based on the different styles created by the design team and this `tablessStyle` can become something like `style` or `tabStyle` and can be an enum